### PR TITLE
Properly expose the promise data in loadRemote

### PR DESCRIPTION
**Pull request by crisbeto**
Fixes https://github.com/rubenv/angular-gettext/issues/238

### CODE REVIEW

File: src/catalog.js [hunk 1]
```diff
@@ -104,16 +104,23 @@ angular.module('gettext').factory('gettextCatalog', function (gettextPlurals, $h
...
+            });
+
+            // note that if we returned $http.get(...).then,
+            // the response wouldn't be available to the next
+            // callback that gets registered
+            promise.then(function (response) {
                 var data = response.data;
                 for (var lang in data) {
                     catalog.setStrings(lang, data[lang]);
                 }
```
Comment (rubenv) [inline src/catalog.js]: just put `return response;` here and it becomes a one-line change. No other changes needed then.

Comment (crisbeto) [inline src/catalog.js]: My concern was that it might not be obvious as to what it does and it could get deleted at some point.

Comment (rubenv) [inline src/catalog.js]: That's why we have a unit test :-)

Comment (crisbeto) [inline src/catalog.js]: There we go.

**Comment by rubenv** [comment]
Great! Merging, thanks a lot!

**Comment by rubenv** [comment]
Released as `v2.1.2`.

**Pull request merged**